sources/tools/Stride.Importer.Assimp/Material/Flags.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation and Contributors (https://dotnetfoundation.org/ & https://stride3d.net) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
```

Comment:
Hmm...I am not sure about that. Because I originaly separated classes/enums from this file:
https://github.com/stride3d/stride/blob/master/sources/tools/Stride.Assimp/MaterialStack.cs

Reply:
If it comes from another file with a license header, forget about my comment then. For other files with no header or for new files, however, use the "simple" one.